memory_view.c

```diff
@@ -326,6 +330,12 @@ rb_memory_view_parse_item_format(const char *format,
         }
     }
 
+    // adjust total size with the alignment size of the first element
```

Comment:
alignment of whole struct should be the maximum alignment of members, not first one.
(Since alignments are always power of 2, the maximum one is LCM of alignment of members.)

Reply:
Ah, I've completely forgot the spec...  Thank you so much for catching this.